go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/libp2p/go-libp2p v0.22.0
 	github.com/libp2p/go-libp2p-kad-dht v0.18.0
 	github.com/libp2p/go-libp2p-kbucket v0.4.7
-	github.com/libp2p/go-libp2p-pubsub v0.8.1-0.20220908052023-8866ca88a105
+	github.com/libp2p/go-libp2p-pubsub v0.8.2-0.20221102045350-1e161006c43f
```

Comment:
We upgrade our libp2p-pubsub to enable the recent refactoring we upstreamed: https://github.com/libp2p/go-libp2p-pubsub/pull/503